djangocms_pageadmin/test_utils/helpers.py

```diff
@@ -0,0 +1,47 @@
+from django.test import RequestFactory
+
+from cms.toolbar.toolbar import CMSToolbar
+
+from djangocms_versioning.cms_toolbars import VersioningToolbar
```

Comment:
Wondering why we're doing anything with VersioningToolbar here. Looks like too much copy and paste